modules/seqexec/server/src/main/scala/seqexec/server/package.scala

```diff
@@ -223,6 +234,11 @@ package object server {
       SeqActionF.embed(ioa)
   }
 
+  implicit class SeqActionOps[F[_]: MonadError[?[_], Throwable], A](a: SeqActionF[F, A]) {
+    def actionF: F[A] =
+      a.leftWiden[Throwable].rethrowT
```

Comment:
We can define this a little more generically by letting the error type be any subtype of throwable.

```scala
implicit class MoreEitherTOps[F[_]: MonadError[?[_], Throwable], A <: Throwable, B](fa: EitherT[F, A, B]) {
    def widenRethrowT: F[B] = 
      fa.leftWiden[Throwable].rethrowT
  } 
```

And with a bit more work to get it to infer right we can generalize it to any subtype of any error type.

```scala
implicit class MoreEitherTOps[F[_], A, B](fa: EitherT[F, A, B]) {
  def widenRethrowT[T](
    implicit me: MonadError[F, T],
             at: A <:< T
  ): F[B] = 
    fa.leftMap(at).rethrowT
} 
```

Reply:
👍